test/SimpleClientTests/ConfigSpec.hs

```diff
@@ -98,8 +107,14 @@ resolveAccountAddressSpec = describe "resolveAccountAddress" $ do
   where r = resolveAccountAddress
         s1 = "35FtQ8HgRShXLGUer7k8wtovjKAcSQ2Ys8RQPx27KfRA7zf7i4"
         s2 = "4RDhNeQB7DUKcKNStBQfLjU6y32HYDMxsJef2ATVncKRYJWoCV"
-        (Right a1) = IDTypes.addressFromText s1
-        (Right a2) = IDTypes.addressFromText s2
+        a1 = case IDTypes.addressFromText s1 of
+          Right addr -> addr
+          -- This does not happen
+          Left err -> error err
+        a2 = case IDTypes.addressFromText s2 of
+          Right addr -> addr
+          -- This does not happen
```

Comment:
None of these comments are useful. State why it does not happen, or just use `fromRight`.